apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs

```diff
@@ -255,17 +255,24 @@ public static SpannerCommandTextBuilder CreateUpdateTextBuilder(string table)
         /// for executing a DDL statement.
         /// </summary>
         /// <param name="ddlStatement">The full DDL statement. Must not be null.</param>
+        /// <param name="extraDdlStatements"></param>
         /// <returns>A <see cref="SpannerCommandTextBuilder"/> representing a <see cref="F:SpannerCommandType.Ddl"/> Spanner command.</returns>
-        public static SpannerCommandTextBuilder CreateDdlTextBuilder(string ddlStatement)
+        public static SpannerCommandTextBuilder CreateDdlTextBuilder(string ddlStatement, string[] extraDdlStatements)
```

Comment:
Would this make sense to be a parameter array? (As it is for CreateDdlCommand)

Reply:
makes sense